modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step12_rgmb_empire.md

```diff
@@ -0,0 +1,129 @@
+# Reactor Geometry Mesh Builder Example: 2-D Empire Microreactor Core Mesh with Control Drum Stitching
+
+This example illustrates the use of RGMB mesh generators to define a heterogeneous 2D hexagonal microreactor core with control drum structures for the EMPIRE problem ([!cite](matthews2021coupled)). The primary focus of this tutorial is to familiarize users with [ControlDrumMeshGenerator.md] to create control drum structures for stitching into the core lattice. For more information about RGMB and how to use it for a conventional fast reactor problem, please see [step10_rgmb.md] and [step11_rgmb_het_hom.md], respectively.
+
+!media tutorial04_meshing/rgmb_empire_stepbystep.png
+       id=tutorial04-rgmb_abtr_stepbystep
+       caption=Visualization of meshing steps to build the 2D EMPIRE core mesh with RGMB.
+       style=width:75%;display:block;margin-left:auto;margin-right:auto;
+
+## ReactorMeshParams
+
+[ReactorMeshParams.md] contains global mesh/geometry parameters including whether the final mesh is 2D or 3D, Cartesian or hexagonal, assembly pitch, and the axial discretization for the final extruded geometry. This information will be accessible to the other RGMB mesh generators and consistently used. Here we also invoke the option to enable flexible assembly stitching with [!param](/Mesh/ReactorMeshParams/flexible_assembly_stitching), so that dissimilar assembly structutes can be stitched into the reactor core without any hanging nodes. This parameter needs to be set to true in order to use [ControlDrumMeshGenerator.md].
```

Comment:
```suggestion
[ReactorMeshParams.md] contains global mesh/geometry parameters including whether the final mesh is 2D or 3D, Cartesian or hexagonal, assembly pitch, and the axial discretization for the final extruded geometry. This information will be accessible to the other RGMB mesh generators and consistently used. Here we also invoke the option to enable flexible assembly stitching with [!param](/Mesh/ReactorMeshParams/flexible_assembly_stitching), so that dissimilar assembly structures can be stitched into the reactor core without any hanging nodes. This parameter needs to be set to true in order to use [ControlDrumMeshGenerator.md].
```